topics/assembly/tutorials/ecoli_comparison/tours/tour.yaml

```diff
@@ -893,6 +893,32 @@ steps:
         <li>"From" to the collection produced at the previous step</li>
       </ul>
     position: left
+  - title: Cutting columns form annotation data
+    content: >-
+      This will create a 21st column containing `0` for all rows. Now we can cut
+      necessary columns from these datasets. These columns are 8 (start), 9
+      (end), 15 (gene symbol), 21 (dummy column we just created), and c10
+      (strand).
+    backdrop: true
+  - title: Cutting columns form annotation data
+    element: '#tool-search-query'
+    content: Search for "Cut columns from a table" tool
+    placement: right
+    textinsert: Cut columns
+  - title: Cutting columns form annotation data
+    element: '#tool-search'
+    content: Click on the "Cut columns from a table" tool to open it
+    placement: right
+    postclick:
+      - 'a[href$="/tool_runner?tool_id=Cut1"] .tool-old-link'
+  - title: Cutting columns form annotation data
+    element: '#tool-search'
+    content: |-
+      Execute the tool with the following parameters<ul>
+        <li>"Cut columns" to `c8,c9,c15,c21,c10`</li>
```

Comment:
Thanks a lot @polkhe , It looks good, just a thing about the column numbers :  your fix cut the columns c8,c9,c15,c21,c10 , but the hand on tutorial cut the columns c8,c9,c14,c19,c10 .  Would you mind verifying the column, and maybe correct the tutorial if they are wrong?

Reply:
Thank you for the comment. There is a mistake in a hands-on section, since in the textual part we state: 

> Now we can cut necessary columns from these datasets. These columns are 8 (start), 9 (end), 15 (gene symbol), 21 (dummy column we just created), and c10 (strand).

I will correct the tutorial

Reply:
tutorial adjusted in this [PR](https://github.com/galaxyproject/training-material/pull/1193/files)